applicationLibrary/molecularDynamics/molecularDynamicsLibrary/AxilrodTellerMutoFunctorHWY.h

```diff
@@ -0,0 +1,1820 @@
+          const auto y2v = loadPacked<alignedSoAView, false>(tag_double, &yptr2[j]);
+          const auto z2v = loadPacked<alignedSoAView, false>(tag_double, &zptr2[j]);
+
+          const auto dx = x2v - x1v;
+          const auto dy = y2v - y1v;
+          const auto dz = z2v - z1v;
+
+          const auto dist2 = highway::MulAdd(dx, dx, highway::MulAdd(dy, dy, dz * dz));
+          const auto cutoffMask = highway::Le(dist2, highway::Set(tag_double, cutoffSquared));
+          const auto sqrtR2 = highway::Sqrt(dist2);
+          const auto r5 = highway::Mul(highway::Mul(dist2, dist2), sqrtR2);
+          const auto invr5 = highway::IfThenElse(cutoffMask, highway::Div(highway::Set(tag_double, 1.0), r5),
+                                                 highway::Zero(tag_double));
+
+          const size_t idx = index(i, j);
+
+          if constexpr (LowerTriangle) {
+            storePacked<false, false>(tag_double, &_dx[idx], dx);
+            storePacked<false, false>(tag_double, &_dy[idx], dy);
+            storePacked<false, false>(tag_double, &_dz[idx], dz);
+            storePacked<false, false>(tag_double, &_squared[idx], dist2);
+            storePacked<false, false>(tag_double, &_invR5[idx], invr5);
+          } else {
+            storePacked<true, false>(tag_double, &_dx[idx], dx);
+            storePacked<true, false>(tag_double, &_dy[idx], dy);
+            storePacked<true, false>(tag_double, &_dz[idx], dz);
+            storePacked<true, false>(tag_double, &_squared[idx], dist2);
+            storePacked<true, false>(tag_double, &_invR5[idx], invr5);
+          }
+        }
+
+        // Remainder
+        if (j < rowLength) {
+          const size_t width = rowLength - j;
+
+          const auto x2v = highway::LoadN(tag_double, &xptr2[j], width);
+          const auto y2v = highway::LoadN(tag_double, &yptr2[j], width);
+          const auto z2v = highway::LoadN(tag_double, &zptr2[j], width);
+
+          const auto dx = x2v - x1v;
+          const auto dy = y2v - y1v;
+          const auto dz = z2v - z1v;
+
+          const auto dist2 = highway::MulAdd(dx, dx, highway::MulAdd(dy, dy, dz * dz));
+          const auto cutoffMask = highway::Le(dist2, highway::Set(tag_double, cutoffSquared));
+          const auto sqrtR2 = highway::Sqrt(dist2);
+          const auto r5 = highway::Mul(highway::Mul(dist2, dist2), sqrtR2);
+          const auto invr5 = highway::IfThenElse(cutoffMask, highway::Div(highway::Set(tag_double, 1.0), r5),
+                                                 highway::Zero(tag_double));
+
+          const size_t idx = index(i, j);
+
+          highway::StoreN(dx, tag_double, &_dx[idx], width);
+          highway::StoreN(dy, tag_double, &_dy[idx], width);
+          highway::StoreN(dz, tag_double, &_dz[idx], width);
+          highway::StoreN(dist2, tag_double, &_squared[idx], width);
+          highway::StoreN(invr5, tag_double, &_invR5[idx], width);
+        }
+      }
```

Comment:
As the remainder is padded, you shouldn't need special case handling for the remainder